pipelines/shopify_dlt_pipeline.py

```diff
@@ -0,0 +1,21 @@
+"""Pipeline to load shopify data into BigQuery.
+"""
+
+import dlt
+from typing import List
+from shopify_dlt import shopify_source
+
+def load(resources: List[str]) -> None:
```

Comment:
if you add start_date let's demonstrate it

Reply:
done

Reply:
where? :)

Reply:
maybe the change is not comitted?

Reply:
ahh ok. it is in the tests. but this script is distributed with the pipeline and used for copy and paste. so we should show how start date is used as well.